go/api/go_api/tests/utils.py

```diff
@@ -1,5 +1,27 @@
 from mock import Mock, patch
 
+from vumi.tests.utils import MockHttpServer
```

Comment:
Could we use `vumi.tests.fake_connection.FakeHttpServer` instead? It's not quite a drop-in replacement, but it's more deterministic and less prone to random network-related failures.